### PR TITLE
Enums for ruby sdk

### PR DESCRIPTION
**Pull request by amahadaya**

### CODE REVIEW

File: docs/AccountType.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Freeclimb::AccountType
+
+## Properties
+
+| Name | Type | Description | Notes |
+| ---- | ---- | ----------- | ----- |
+
```
Comment (ngarg-vail) [inline docs/AccountType.md:9]: This should contain the enum property values. Looking at the rest of the enum based docs this seems to be a consistent error

File: spec/models/language_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Unit tests for Freeclimb::Language
+# Automatically generated by openapi-generator (https://openapi-generator.tech)
+# Please update as you see appropriate
+describe Freeclimb::Language do
+  let(:instance) { Freeclimb::Language.new }
+
+  describe 'test an instance of Language' do
+    it 'should create an instance of Language' do
+      expect(instance).to be_instance_of(Freeclimb::Language)
```
Comment (ngarg-vail) [inline spec/models/language_spec.rb:25]: General comment for enum based models - can we have any tests around the enum key/values?

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

File: docs/AccountStatus.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Freeclimb::AccountStatus
+
+## Properties
+
+| Name | Type | Description | Notes |
+| ---- | ---- | ----------- | ----- |
```
Comment (ngarg-vail) [inline docs/AccountStatus.md:6]: There is still not documentation with the enum values.

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

File: docs/AccountStatus.md (hunk 3 above)
Comment (amahadaya) [reply docs/AccountStatus.md:6]: Is this good for the enum documentation structure:

# Freeclimb::AccountType

## Properties

| Name | Type | Description | Notes |
| ---- | ---- | ----------- | ----- |
| **TRIAL** | **AccountType** |  | Represented in Ruby as "trial" |
| **FULL** | **AccountType** |  | Represented in Ruby as "full" |

## Example

```ruby
require 'freeclimb'

instance = Freeclimb::AccountType::[NAME OF ENUM]
```

Comment (ngarg-vail) [reply docs/AccountStatus.md:6]: thats a great format. lets apply that every where

Comment (amahadaya) [reply docs/AccountStatus.md:6]: Updated in latest commit

File: spec/models/language_spec.rb (hunk 2 above)
Comment (amahadaya) [reply spec/models/language_spec.rb:25]: This is done in latest commit

**Review by ngarg-vail — APPROVED** [review]

**Review by ngarg-vail — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,166 +1,248 @@
...
 None
-<a name-"4.0.4"></a>
+
+<a name="4.1.0"></a>
+
+## [4.1.0] - 2023-01-06
+
+### Changed
+
+- Enums have been implemented in the csharp sdk
```
Comment (ngarg-vail) [inline CHANGELOG.md]: wrong sdk

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

**Review by ngarg-vail — APPROVED** [review]

**Pull request merged**